examples/2_counter/dune

```diff
@@ -0,0 +1,6 @@
+(executable
+  (name Main)
+  (public_name 2_counter)
+  (package ReWeb)
+  (libraries ReWeb lwt str)
+  (preprocess (pps lwt_ppx)))
```

Comment:
Oh btw we don't need the `str` library and the `lwt_ppx` preprocessor in these examples so far, they can be removed.